packages/eslint-custom-rules/no-tailwind-colors.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+    meta: {
+      type: 'suggestion',
+      docs: {
+        description: 'Disallow Tailwind CSS colors in classNames',
+        category: 'Best Practices',
+        recommended: true,
+      },
+      schema: [],
+    },
+    create: function (context) {
+      const tailwindColors = [
+        'gray-', 'red-', 'yellow-', 'green-', 'blue-', 'indigo-', 'purple-', 'pink-',
+        'white-', 'black-',
+      ];
+  
+      function isTailwindColor(value) {
+        return tailwindColors.some((color) => value.startsWith(`text-${color}`) || value.startsWith(`bg-${color}`));
```

Comment:
not sure if this is the best approach because we'd also have border, shadow, etc 🤔